wifimgr.py

```diff
@@ -6,6 +6,7 @@
 ap_ssid = "WifiManager"
 ap_password = "tayfunulu"
 ap_authmode = 3  # WPA2
+connectToOpenWifis = False
```

Comment:
please stay with the style you see (e.g. 1 line above).

so, no javaLikeCamelCase, but rather python_style for variable names.